### PR TITLE
enhancement/logo-fade

### PR DESCRIPTION
**Pull request by ArleyAV**
enhancement/logo-fade

1. Fix logo entrance when it scrolls down and blue navbar displays.

image
image
image
image

### CODE REVIEW

**Comment by josemariogutierrez** [comment]
Plase make the logo appear with an fade in effect, right now is just appearing right away. Also, please make it appear when the big logo in the hero is starting to be covered by the navbar, the idea is to not display 2 logos at the same time in the homepage.

**Comment by ArleyAV** [comment]
@josemariogutierrez I already added a fade-in transition to logo in navbar.

**Review by jhonatanrojasbastidas — APPROVED** [review]

File: css/style.css [hunk 1]
```diff
@@ -54,7 +54,13 @@ video {
...
 	position: absolute;
 	top: 11px;
-	left: 20px;
+  left: 20px;
+  transition: opacity .2s ease-in;
+  opacity: 0;
+}
+
+.main-nav--colored .main-nav__logo{
+ opacity: 1;
```
Comment (camigomezdev) [inline css/style.css]: Please, give format to all file

**Comment by ArleyAV** [comment]
@josemariogutierrez 

I fixed the navbar background transition, now it appears when the users scroll down and  the hero logo disappears out of the screen.

File: js/main_nav.js [hunk 2]
```diff
@@ -6,6 +6,11 @@ let navBurguer = document.getElementById("main-nav__burguer");
 
 let overlayMenu = document.getElementById("main-nav__menu-items");
 
+let logoElement = document.querySelector(".homepage-hero__logo");
+let logoHeight = logoElement.offsetHeight;
+let logoTop = logoElement.offsetTop;
+
+
```
Comment (christian-cg) [inline js/main_nav.js:16]: What is this space for?

File: js/main_nav.js [hunk 3]
```diff
@@ -17,6 +22,14 @@ navBurguer.addEventListener("click", function () {
...
+  window.scrollY >= logoHeight + logoTop ? overlay.classList.add("main-nav--colored") : overlay.classList.remove("main-nav--colored");
+});
+
 window.addEventListener("scroll", function () {
-  window.scrollY >= 80 ? overlay.classList.add("main-nav--colored") : overlay.classList.remove("main-nav--colored");
-});
\ No newline at end of file
+  window.scrollY >= logoHeight + logoTop ? overlay.classList.add("main-nav--colored") : overlay.classList.remove("main-nav--colored");
+});
+
```
Comment (christian-cg) [inline js/main_nav.js]: What is this space for?

File: js/main_nav.js [hunk 4]
```diff
@@ -6,6 +6,11 @@ let navBurguer = document.getElementById("main-nav__burguer");
 
 let overlayMenu = document.getElementById("main-nav__menu-items");
 
+let logoElement = document.querySelector(".homepage-hero__logo");
+let logoHeight = logoElement.offsetHeight;
+let logoTop = logoElement.offsetTop;
```
Comment (christian-cg) [inline js/main_nav.js]: I believe this variable might be named a little more explanatory, like `logoOffsetTop`, what do you think? Something that help us understand the variable faster.

**Review by christian-cg — COMMENTED** [review]
Looks good !

**Comment by Frankruiz90** [comment]
I do not know if the idea is that when you click on the down button on the top

image

File: css/style.css [hunk 5]
```diff
@@ -54,7 +54,13 @@ video {
...
 }
 
 .main-nav__logo__image {
```
Comment (hanna446) [inline css/style.css]: I think the class name is incorrect, a better approach could be `main-nav__logo-image`

**Comment by Frankruiz90** [comment]
you have conflicts

File: css/style.css (hunk 5 above)
Comment (camigomezdev) [reply css/style.css]: Arley didn't add or modify this line... He should not fix it.

**Comment by ArleyAV** [comment]
@christiang6 
I already added the explanations about the code you asked.

File: js/main_nav.js [hunk 6]
```diff
@@ -17,6 +25,16 @@ navBurguer.addEventListener("click", function () {
   }
 });
 
+// This function recalculates the values of logo's height and the space between logo and the very top of the viewport when a user makes resize of the window
```
Comment (jusk02) [inline js/main_nav.js]: please reference the styleguide to document your code
https://github.com/airbnb/javascript#comments

**Comment by jusk02** [comment]
In general the code seems good, just some format/style issues.
Resuming other peoples comments:
- please indent your code, Prettier can make that an easy task
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

- Update your documentation/comments to match the style guide
https://github.com/airbnb/javascript#comments

- Remove unneeded extra white lines, and add expected white spaces(prettier does whiete spaces too if you install it)
https://github.com/airbnb/javascript#whitespace
https://github.com/airbnb/css#formatting

Reach out to any tutor if you have any doubts

**Review by Frankruiz90 — APPROVED** [review]

**Review by camigomezdev — CHANGES_REQUESTED** [review]
Por favor arregla los ultimos comentarios

**Review by camigomezdev — APPROVED** [review]

**Pull request merged**